src/Mod/Sketcher/App/SketchObject.cpp

```diff
@@ -2291,6 +2291,27 @@ int SketchObject::trim(int GeoId, const Base::Vector3d& point)
         return false;
 
     };
+    
+    // this is a helper function to remove Equal constraints from Line segments. 
+    auto delEqualConstraintsOnGeoId = [this] (int GeoId) {

```

Comment:
I might actually spell out "delete" here (or change to "remove") -- my brain read it as "delta" first.

Reply:
Yes, I agree. But the convention in the file (and the method) seems to be to use the prefix _del_. I.e _delConstraint_ , _delConstraints_ and  _delConstraintOnPoint_ etc. I don't mind either way.

Reply:
OK, sorry, I did not look at the rest of the file. I think go with the convention, then.